src/storm/modelchecker/prctl/helper/SparseMdpPrctlHelper.cpp

```diff
@@ -379,7 +379,9 @@ SparseMdpHintType<ValueType> computeHints(Environment const& env, SolutionType c
     if (!result.eliminateEndComponents) {
         extractValueAndSchedulerHint(result, transitionMatrix, backwardTransitions, maybeStates, selectedChoices, hint, result.uniqueSolution);
     } else {
-        STORM_LOG_WARN_COND(hint.isEmpty(), "A non-empty hint was provided, but its information will be disregarded.");
+        if (hint.isEmpty()) {
+            STORM_LOG_TRACE("Warn A non-empty hint was provided, but its information will be disregarded.");
+        }
     }
```

Comment:
What is the reason for this change? Too much output otherwise?

Reply:
Exactly, in particular because it is not really something a user should be concerned about.